crates/tests/sawtooth_publisher/src/main.rs

```diff
@@ -63,31 +59,45 @@ pub async fn main() -> Result<(), Box<dyn std::error::Error>> {
         height * iterations
     );
     let c8y_msg = Topic::new("tedge/measurements")?;
-    let c8y_cmd = Topic::new("c8y/s/ds")?;
```

Comment:
Don't we need to subscribe to this topic?

Reply:
No this topic is unused. The sawtooth publisher has been written after an example from the `mqtt_client` crate.
It would be good to clean it up. This PR does just the minimum to remove this deprecated dependency.